services/BBB.go

```diff
@@ -182,7 +182,7 @@ func applyRules(server s.Server, participantsCount int, bbb BBBService) s.ScaleR
 	}
 
 	// Rule 2 memory
-	if memMaxUsageDelta := server.ServerRamUsage - bbb.Config.Resources.Memory.MaxUsage; memMaxUsageDelta > 0 && server.ServerRam < int32(bbb.Config.Resources.Memory.MaxBytes) {
+	if memMaxUsageDelta := server.ServerRamUsage - bbb.Config.Resources.Memory.MaxUsage; memMaxUsageDelta > 0 && server.ServerRam <= int32(bbb.Config.Resources.Memory.MaxBytes) {
```

Comment:
I don't think this is right, this would generate proposals to scale up by 0 CPUs/bytes when a server is scaled up to the configured max and has high load